utils/score.ts

```diff
@@ -0,0 +1,33 @@
+import {EntityOccurrence} from "~/utils/entities/occurrence";
+import Occurrence = EntityOccurrence.Occurrence;
+
+const commonDishes = ["pizza", "suppe", "salat", "bowl", "kühltheke"]
+
+function scoreOccurrence(occ: Occurrence): number {
+    let score = 10;
+
+    const reviewData = occ.dish.reviewData;
+
+    // Bump occurrences with more reviews
+    score += reviewData.metadata.reviewCount;
+
+    // Bump occurrences with any image
+    if (reviewData.reviews.find(r => r.images.length > 0))
+        score += 1;
+
+    // Punish "always" available dishes
+    if (commonDishes.find(c => occ.dish.nameDe.toLowerCase().includes(c)))
```

Comment:
Please use .some instead of .find - does the same in this case but faster and semantically correct.